### PR TITLE
Adding third-party integration info

### PR DESCRIPTION
**Pull request by dke-benjy**
Would like a review on if this is an acceptable way to link out to our partner integrated solutions in market.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/double-key-encryption.md [hunk 1]
```diff
@@ -564,3 +565,15 @@ If you're an Office Insider, DKE is enabled for you. Otherwise, enable DKE for y
 If you want, once you're finished setting up DKE, you can migrate content that you've protected using HYOK labels to DKE labels. To migrate, you'll use the AIP scanner. To get started using the scanner, see [What is the Azure Information Protection unified labeling scanner?](/azure/information-protection/deploy-aip-scanner).
 
 If you don't migrate content, your HYOK protected content will remain unaffected.
+
+## Other Deployment Options
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
## Other deployment options
```

File: microsoft-365/compliance/double-key-encryption.md [hunk 2]
```diff
@@ -564,3 +565,15 @@ If you're an Office Insider, DKE is enabled for you. Otherwise, enable DKE for y
...
+
+We realize that for some customers in highly regulated industries, this standard reference implementation using software based keys may not be sufficient to meet their enhanced compliance obligations and needs.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
We realize that for some customers in highly regulated industries, this standard reference implementation using software-based keys may not be sufficient to meet their enhanced compliance obligations and needs.
```

File: microsoft-365/compliance/double-key-encryption.md [hunk 3]
```diff
@@ -564,3 +565,15 @@ If you're an Office Insider, DKE is enabled for you. Otherwise, enable DKE for y
...
+We have partnered with various third-party Hardware Security Module (HSM) vendors to bring support for enhanced key management options to the Double Key Encryption Service:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
We have partnered with various third-party hardware security module (HSM) vendors to bring support for enhanced key management options to the DKE service, including:
```

File: microsoft-365/compliance/double-key-encryption.md [hunk 4]
```diff
@@ -564,3 +565,15 @@ If you're an Office Insider, DKE is enabled for you. Otherwise, enable DKE for y
...
+
+ - [Entrust](https://www.entrust.com/digital-security/hsm/services/packaged-services/double-key-encryption-integration#:~:text=Entrust%20Double%20Key%20Encryption%20for%20Microsoft%20AIP%2C%20offered,trust%20for%20the%20protection%20of%20sensitive%20cryptographic%20keys.) 
+
+- [Thales](https://cpl.thalesgroup.com/cloud-security/encryption/double-key-encryption) 
+
+Please reach out directly to these vendors for more information and guidance on their in-market DKE HSM solutions.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
Reach out directly to these vendors for more information and guidance on their in-market DKE HSM solutions. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review only - @dke-benjy, kindly commit my suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@dke-benjy Gentle reminder

**Comment by yogkumgit** [comment]
@dke-benjy Please update the PR with suggested changes, if you agree. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**